_data/simple-icons.json

```diff
@@ -2389,7 +2389,7 @@
         {
             "title": "Deno",
             "hex": "000000",
-            "source": "https://github.com/denoland/deno_website2/blob/892750b99cc260f8b1b2a4bcea59eab9101dd67c/public/logo.svg"
+            "source": "https://github.com/denoland/dotland/blob/717b6815c95b413336bb2bf1ee9177c120eb894c/public/logo.svg"
```

Comment:
Great work, this is actually a perfect match! Can you just add `"license": "MIT"` here?

Reply:
License added. 

> Also does this icon use the changes from your latest commit on denoland/dotland?

Yes. Source link updated.